### ISSUE TITLE
User stories Ahmad, Saeeda, Zahra

### ISSUE THREAD

**Issue by ionpaani** [ionpaani/Sprint2andProject#1]
USER STORIES AHMAD SAEEDA ZAHRA

User story Ahmad: As a clinical geneticist , I want to regularly review and revise variants of unclassified significance  
so that I can provide my patients with a result.  
 
User story Saeeda: As an educator in genomics, I want to teach students the importance of using the correct transcript and genomic built to get the most accurate information to classify a variant.  

User story Zahra: As a researcher in diabetes I face a number of monogenic diabetes cases which can generate variants of unclassified significance, I want to analyse these variants on a regular basis in order to get them most updated classification of these variants so that I can progress in my research. 
 
All three user stories would essentially need the same program. We used ChatGPT and entered the following information win Laymans terms: 

**PROMPT FOR CHATGPT**
I want you to write me code based on the following information   

1. I will give you a variant.   
2. Check the variant in variant validator and get the most recent transcript and use the result from variant validator transcript and put it in VEP and get the information for that variant.   
3. Give me code that i can put into Pycharm which is within Ubuntu Linux  """

**Comment by ionpaani** [comment]
Following on from this. 

**Summary**
Ahmad created the skeleton code using Chat-GPT. He then altered each part step-wise using various resources such as YouTube, Google, StackOverflow, ChatGPT.  From this he created a working code which he is going to assign tasks. 

**Team members**
Ahmad - Project Lead
Saeeda
Zahra
Violetta

**Comment by al-baba** [comment]
can close as text moved to frontpage (README.md)
